### PR TITLE
Adding yml for Linux builds

### PR DESCRIPTION
**Pull request by anmenaga**
## PR Summary

As part of improving the build process, the UI build pipeline for Linux Builds was converted to a yml-based pipeline.

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [X] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [X] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: tools/releaseBuild/LinuxBuild.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+       {
+           $vstsCommandString = "vso[task.setvariable variable=SkipFxDependent]true"    
+       }
+       else
+       {
+           $vstsCommandString = "vso[task.setvariable variable=SkipFxDependent]false"
+       }
+       
+       Write-Verbose -Message "$vstsCommandString " -Verbose
+        Write-Host -Object "##$vstsCommandString"
```
Comment (adityapatwardhan) [inline tools/releaseBuild/LinuxBuild.yml]: Fix indentation.

**Review by adityapatwardhan — APPROVED** [review]

File: tools/releaseBuild/LinuxBuild.yml (hunk 1 above)
Comment (anmenaga) [reply tools/releaseBuild/LinuxBuild.yml]: fixed.

File: tools/releaseBuild/LinuxBuild.yml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      else
+      {
+          $vstsCommandString = "vso[task.setvariable variable=SkipFxDependent]false"
+      }
+
+      Write-Verbose -Message "$vstsCommandString " -Verbose
+      Write-Host -Object "##$vstsCommandString"
+    displayName: 'Skip FxDependent for PS v6.0.*'
+
+  - task: ShellScript@2
```
Comment (TravisEz13) [inline tools/releaseBuild/LinuxBuild.yml]: This task shouldn't be needed anymore

File: tools/releaseBuild/LinuxBuild.yml [hunk 3]
```diff
@@ -0,0 +1,122 @@
+resources:
+- repo: self
+  clean: true
+jobs:
+
+- job: Job_1
```
Comment (TravisEz13) [inline tools/releaseBuild/LinuxBuild.yml]: Give the job a meaningful name, like `Build`

**Review by TravisEz13 — APPROVED** [review]
2 minor issues.

File: tools/releaseBuild/LinuxBuild.yml (hunk 2 above)
Comment (anmenaga) [reply tools/releaseBuild/LinuxBuild.yml]: good point. fixed.

File: tools/releaseBuild/LinuxBuild.yml (hunk 3 above)
Comment (anmenaga) [reply tools/releaseBuild/LinuxBuild.yml]: updated.

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
maybe move the file to a subfolder (in the same location as the file) called `azureDevOps`

**Pull request merged**